powerauth-push-server/src/main/java/io/getlime/push/service/batch/storage/CampaignMessageStorage.java

```diff
@@ -28,7 +28,7 @@
  *
  * @author Petr Dvorak, petr@wultra.com
  */
-public class CampaignMessageStorageMap implements ItemStorageMap<Long, PushCampaignEntity> {
+public class CampaignMessageStorage implements ItemStorage<Long, PushCampaignEntity> {
```

Comment:
🤔 Should we set expiration even for campaigns?

Reply:
Ok, let's ignore expiration here because the campaign is cached for a single run only.